src/app/Dice.ts

```diff
@@ -0,0 +1,47 @@
+import { GetRandomValue } from './Interfaces/DiceInterface';
+import { AnimalNames } from './Enums/AnimalNamesEnum';
+
+class Dice implements GetRandomValue {
+  constructor(private diceSides: AnimalNames[]) {}
+  getRandomValue(): AnimalNames {
+    return this.diceSides[Math.floor(Math.random() * 12)];
```

Comment:
Consider using lodash and its 'sample' method.

Reply:
Ok, thanks a lot for your suggestion!

Reply:
You can also consider shuffling array in constructor for better randomization. There is function for that in lodash named 'shuffle'. But let's leave it for future refactor.